extension/js/common/helpers.ts

```diff
@@ -13,7 +13,7 @@ import { PassphraseStore } from './platform/store/passphrase-store.js';
 import { Bm } from './browser/browser-msg.js';
 import { PgpPwd } from './core/crypto/pgp/pgp-password.js';
 
-export const isFesUsed = async (acctEmail: string) => {
+export const isCustomUrlFesUsed = async (acctEmail: string) => {
```

Comment:
Also, let's change everywhere that is currently `CustomUrlFes` to become `CustomerUrlFes` I think that makes the name clearer (the customer runs that FES themselves)

Reply:
ok